bin/pping.py

```diff
@@ -27,29 +27,37 @@
 import signal
 import getopt
 import time
+import logging
 
 import nav.daemon
 from nav.daemon import safesleep as sleep
+from nav.logs import init_generic_logging, convert_debug_level_to_loglevel
 from nav.statemon import statistics
 from nav.statemon import megaping
 from nav.statemon import db
 from nav.statemon import config
 from nav.statemon import circbuf
-from nav.statemon import debug
 from nav.statemon.event import Event
 from nav.statemon.netbox import Netbox
 
 
+LOGGER = logging.getLogger('bin.pping')
```

Comment:
I would much rather this be under the `nav` namespace, even though much of the functionality of pping has been stuffed into the `bin/pping.py` script. It doesn't seem natural to set log levels for `bin.pping` in `logging.conf`.